labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -37,6 +41,18 @@ object OptionExercise02 {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+    rooms.values.map((a)=> maybeStringToInt(a)).sum
+  }
+  
+  def maybeStringToInt(int:Option[String]): Int = {
+    int match{
+      case Some(string) => try{
+        string.toInt
+      } catch {
+        case e:Exception => 0
+      }
```

Comment:
It's interesting that you used a try catch exception here. I didn't even think to use it anywhere throughout the assignment. It definitely looks a lot more readable or natural than just listing a bunch of case statements like I did.